packages/imageviewer-extension/src/index.ts

```diff
@@ -25,6 +25,12 @@ namespace CommandIDs {
 
   export
   const zoomOut = 'imageviewer:zoom-out';
+
+  export
+  const rot90 = 'imageviewer:rot90';
```

Comment:
Maybe we could name the command IDs `imageviewer:rotate-clockwise` and `imageviewer:rotate-counterclockwise` for better human-readability?